assets/js/script.js

```diff
@@ -0,0 +1,162 @@
+// ###### PSEUDO CODE ######
+/*
+start game button and instructions
+- initialize game -
+create new grid (bigger grid each level)
+color grid - one tile with abnormal color (increase color difficulty each lvl)
+set timer of 10 seconds
+if player clicks the abnormal tile, level is passed
+proceed to next level
+if player clicks the wrong tile, will not proceed to next level
+if timer runs out game over
+- end game -
+produce final score
+*/
+// #########################
+
+document.addEventListener('DOMContentLoaded', function () {
+  var score = 0
+  var timeLeft = 30
+  var noOfRows = 2
+  var ranGrid
+  var isGameOver = false
+  var wholeGrid = document.querySelectorAll('.container')
+  var palette = [
```

Comment:
maybe need to comment this code a bit more, or explain it better in your readme. 

qn pointers:
- What does each of the array item represents?
- Why array?
- Does the position of each array item matters?